.github/workflows/ci-presubmit.yaml

```diff
@@ -0,0 +1,239 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Generated by dev/tasks/generate-github-actions
+
+name: ci-presubmit
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
Do we want to reopen PRs? That could make the presubmit merge conflict become worse since reopen won't rebase.